system/setup.py

```diff
@@ -139,6 +171,9 @@ def main():
     global module
     module = AnsibleModule(
         argument_spec = dict(
+            gather_subset=dict(default="all", required=False, type='list'),
```

Comment:
Since this is now type list, it'll be clearer to make the default a list as well:
``` python
             gather_subset=dict(default=["all"], required=False, type='list'), 
```